extensions/noteplan-3/src/today.tsx

```diff
@@ -1,11 +1,11 @@
-import { NoteType } from "./lib/note-utilities";
+import { NoteType, preferences } from "./lib/note-utilities";
 import { NoteDetail } from "./lib/components";
 import { format } from "date-fns";
 
 export default () => (
   <NoteDetail
     entry={{
-      relativePath: `Calendar/${format(new Date(), "yyyyMMdd")}.txt`,
+      relativePath: `Calendar/${format(new Date(), "yyyyMMdd")}.${preferences.extension}`,
```

Comment:
Do we know if these preferences are being backfilled with the default for users who already had the extension?